src/cli/project/export/conda.rs

```diff
@@ -0,0 +1,198 @@
+use std::path::PathBuf;
+
+use clap::Parser;
+
+use itertools::Itertools;
+use miette::IntoDiagnostic;
+use rattler_conda_types::{MatchSpec, Platform};
+
+use crate::cli::LockFileUsageArgs;
+use crate::lock_file::UpdateLockFileOptions;
+use crate::utils::conda_environment_file::{CondaEnvDep, CondaEnvFile};
+use crate::{project, HasFeatures, Project};
+
+// enum to select version spec formatting
+#[derive(clap::ValueEnum, Clone, Debug)]
+pub enum VersionSpec {
+    Manifest,
+    Locked,
+    None,
+}
+
+/// Exports a projects dependencies as an environment.yml
+///
+/// The environment is printed to standard out
+#[derive(Debug, Parser)]
+#[clap(arg_required_else_help = false)]
+pub struct Args {
+    /// The platform to list packages for. Defaults to the current platform.
+    #[arg(long)]
```

Comment:
```suggestion
    #[arg(short, long)]
```
To align with other cli's.